internal/controller/node_reconciler.go

```diff
@@ -5,6 +5,7 @@ import (
 	"time"
 
 	pdbautoscaler "github.com/azure/eviction-autoscaler/api/v1"
+	"github.com/azure/eviction-autoscaler/internal/metrics"
 	"github.com/azure/eviction-autoscaler/internal/podutil"
 	corev1 "k8s.io/api/core/v1"
```

Comment:
can be sepeate pr but we import the same thing twice. :) was wonderign how it changed below.